libs/vue-plugin/src/webpack.ts

```diff
@@ -194,3 +194,22 @@ export function modifyFilenameHashing(
     })
   );
 }
+
+export function modifyTypescriptAliases(
+  config,
+  options: BrowserBuilderSchema,
+  context: BuilderContext
+) {
+  const tsConfigPath = getSystemPath(
+    join(normalize(context.workspaceRoot), options.tsConfig)
+  );
+  config.resolve.alias.delete('@');
+  config.resolve
+    .plugin('tsconfig-paths')
+    // eslint-disable-next-line @typescript-eslint/no-var-requires
+    .use(require('tsconfig-paths-webpack-plugin'), [
```

Comment:
What version of `tsconfig-paths-webpack-plugin` will this resolve? If we add that as a dependency will it resolve our installed version? I have the same concern [here](https://github.com/ZachJW34/nx-plus/blob/master/libs/vue-plugin/src/webpack.ts#L129).

Reply:
From what I'm reading, it will use the version we install. From [this article](https://dev.to/open-wc/nested-dependencies-in-frontend-558c):
```
Basically, node looks into every node_modules folder, starting in the module's parent directory and moving up the file tree, until it finds a directory name which matches the module specifier
```
I believe the dependency is currently being pulled from `@nrwl/web` but I don't think there is a guarantee that it will be that one. I'll add it as a dependency.